conf/messages

```diff
@@ -102,23 +102,17 @@ pages.completionCapacity.other.error                            = Enter your rel
 pages.completionCapacity.error.invalidChars                     = Tell us your relationship to the company using only letters, numbers, hyphens and apostrophes
 
 ## Trading Name Page
-pages.tradingName.description                                   = Will the company trade under another name?
-pages.tradingName.lede                                          = Tell us if you''ll trade using a different name to {0}.
+pages.tradingName.description                                   = Does or will the company trade using a different name?
+pages.tradingName.lede                                          = Tell us if the company will use a trading name that''s different from {0}.
 pages.tradingName.tradingNameLabel                              = Enter the trading name
-pages.tradingName.nameRulesSummary                              = What are the rules for trading names?
-pages.tradingName.nameRulesIntro                                = Companies can trade using a different name to the name registered with Companies House.
-pages.tradingName.nameRulesIntro1                               = It can''t be the same as another company''s.
-pages.tradingName.nameRulesIntro2                               = Check if your trading name is already taken.
-pages.tradingName.nameRulesIntro3                               = Trading names can''t:
-pages.tradingName.nameRules1                                    = include ''limited'', ''Ltd'', ''limited liability partnership, ''LLP'', ''public limited company'' or ''plc''
-pages.tradingName.nameRules2.1                                  = be the same as an
-pages.tradingName.nameRules2.2                                  = existing trade mark
-pages.tradingName.nameRules3.1                                  = contain a ''sensitive'' word or expression unless you
-pages.tradingName.nameRules3.2                                  = get permission
+pages.tradingName.nameRulesSummary                              = What is a trading name?
+pages.tradingName.nameRulesIntro                                = A company can trade using a name that’s different from its registered name. This is also known as a ‘business name’.
+pages.tradingName.nameRulesIntro1                               = Some companies choose a different trading name to help with branding or getting a domain name for their website.
 pages.tradingName.errorQuestion                                 = Enter the trading name
 pages.tradingName.error                                         = Tell us if the company will trade under another name
 pages.tradingName.error.length                                  = Enter a trading name using 35 characters or less
 pages.tradingName.error.invalidChars                            = Tell us the trading name using only letters, numbers and - , . ( ) / & '' !
+pages.tradingdetails.input.hint-text                            = You can't include ‘limited’, ‘Ltd’, ‘limited liability partnership, ‘LLP’, ‘public limited company’ or ‘plc’
```

Comment:
Doesn't look like the apostrophe was escaped here - is it definitely there in the view?